stable/kube-ops-view/values.yaml

```diff
@@ -39,11 +39,15 @@ rbac:
 ## ref: https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/
 podLabels: {}
 
+# Add securityContext
+securityContext: {}
+
+# Defaults for Redis
 redis:
   enabled: false
   usePassword: false
   cluster:
     enabled: false
   master:
     persistence:
-      enabled: false
+      enabled: false
```

Comment:
The end of file new line has been removed.

Reply:
Ok, now there is only one line.